BaseTools/Conf/tools_def.template

```diff
@@ -816,6 +828,142 @@ NOOPT_VS2022_AARCH64_DLINK_FLAGS   = /NOLOGO /NODEFAULTLIB /IGNORE:4001 /IGNORE:
+*_VS2026_IA32_DLINK_XIPFLAGS    = /ALIGN:32
+
+##################
+# X64 definitions
+##################
+*_VS2026_X64_CC_PATH       = DEF(VS2026_BIN_X64)\cl.exe
+*_VS2026_X64_PP_PATH       = DEF(VS2026_BIN_X64)\cl.exe
+*_VS2026_X64_APP_PATH      = DEF(VS2026_BIN_X64)\cl.exe
+*_VS2026_X64_VFRPP_PATH    = DEF(VS2026_BIN_X64)\cl.exe
+*_VS2026_X64_ASLCC_PATH    = DEF(VS2026_BIN_X64)\cl.exe
+*_VS2026_X64_ASLPP_PATH    = DEF(VS2026_BIN_X64)\cl.exe
+*_VS2026_X64_ASM_PATH      = DEF(VS2026_BIN_X64)\ml64.exe
+*_VS2026_X64_SLINK_PATH    = DEF(VS2026_BIN_X64)\lib.exe
+*_VS2026_X64_DLINK_PATH    = DEF(VS2026_BIN_X64)\link.exe
+*_VS2026_X64_ASLDLINK_PATH = DEF(VS2026_BIN_X64)\link.exe
+
+  DEBUG_VS2026_X64_CC_FLAGS     = /nologo /c /WX /GS /W4 /Gs32768 /D UNICODE /O1b2s /GL /Gy /FIAutoGen.h /EHs-c- /GR- /GF /Z7 /Gw /volatileMetadata-
+RELEASE_VS2026_X64_CC_FLAGS     = /nologo /c /WX /GS /W4 /Gs32768 /D UNICODE /O1b2s /GL /Gy /FIAutoGen.h /EHs-c- /GR- /GF /Gw /volatileMetadata-
+NOOPT_VS2026_X64_CC_FLAGS       = /nologo /c /WX /GS /W4 /Gs32768 /D UNICODE /Gy /FIAutoGen.h /EHs-c- /GR- /GF /Z7 /Od /volatileMetadata-
+
+  DEBUG_VS2026_X64_ASM_FLAGS    = /nologo /c /WX /W3 /Cx /Zd /Zi
+RELEASE_VS2026_X64_ASM_FLAGS    = /nologo /c /WX /W3 /Cx /Zd
+NOOPT_VS2026_X64_ASM_FLAGS      = /nologo /c /WX /W3 /Cx /Zd /Zi
+
+  DEBUG_VS2026_X64_NASM_FLAGS   = -Ox -f win64 -g
+RELEASE_VS2026_X64_NASM_FLAGS   = -Ox -f win64
+NOOPT_VS2026_X64_NASM_FLAGS     = -O0 -f win64 -g
+
+# Linker warning 4210 is disabled globally, because it checks if static initializers are present and the VCRuntime is
+# linked. We do not link the VCRuntime (except for HOST_APPLICATIONs) so this warning can be generated erroneously
+# whenever there are static initializers, because we will always fail the VCRuntime linking check
+  DEBUG_VS2026_X64_DLINK_FLAGS  = /NOLOGO /NODEFAULTLIB /IGNORE:4001 /IGNORE:4281 /OPT:REF /OPT:ICF=10 /MAP /ALIGN:32 /SECTION:.xdata,D /SECTION:.pdata,D /Machine:X64 /LTCG /DLL /ENTRY:$(IMAGE_ENTRY_POINT) /SUBSYSTEM:EFI_BOOT_SERVICE_DRIVER /SAFESEH:NO /BASE:0 /DRIVER /DEBUG /ALIGN:4096 /DLL /WX /IGNORE:4210
+RELEASE_VS2026_X64_DLINK_FLAGS  = /NOLOGO /NODEFAULTLIB /IGNORE:4001 /IGNORE:4281 /IGNORE:4254 /OPT:REF /OPT:ICF=10 /MAP /ALIGN:32 /SECTION:.xdata,D /SECTION:.pdata,D /Machine:X64 /LTCG /DLL /ENTRY:$(IMAGE_ENTRY_POINT) /SUBSYSTEM:EFI_BOOT_SERVICE_DRIVER /SAFESEH:NO /BASE:0 /DRIVER /MERGE:.rdata=.data /ALIGN:4096 /DLL /WX /IGNORE:4210
+NOOPT_VS2026_X64_DLINK_FLAGS    = /NOLOGO /NODEFAULTLIB /IGNORE:4001 /IGNORE:4281 /OPT:REF /OPT:ICF=10 /MAP /ALIGN:32 /SECTION:.xdata,D /SECTION:.pdata,D /Machine:X64 /LTCG /DLL /ENTRY:$(IMAGE_ENTRY_POINT) /SUBSYSTEM:EFI_BOOT_SERVICE_DRIVER /SAFESEH:NO /BASE:0 /DRIVER /DEBUG /ALIGN:4096 /DLL /WX /IGNORE:4210
+
+# BaseTools cannot handle XIP modules with different section and file alignment
+*_VS2026_X64_DLINK_XIPFLAGS     = /ALIGN:32
+
+#####################
+# AARCH64 definitions

```

Comment:
TBH, I would be of the opinion that we don't add the MSVC ARM64 definitions to edk2 for VS2026. You can't build the repo with MSVC ARM64 because none of the asm files are in armasm64 format. For awhile we had a downstream workaround for this (essentially porting them over, but even that was a cludge). edk2 has new directly compiled PE target for ARM64 in clangpdb that actually works. Might as well not pollute this with toolchains that don't work and aren't tested. If any users come forward saying they want it and want to make it work, they are welcome to contribute.